assets/templates/census-first-results.tmpl

```diff
@@ -0,0 +1,119 @@
+<section class="ons-hero ons-grid--gutterless ons-hero--census">
+    <div class="ons-hero__container ons-container">
+        <div class="ons-hero__details ons-hero__details--ondark ons-grid__col ons-col-6@m">
+            <header>
+                <h1 class="ons-u-fs-xxxl">{{ localise "Census" .Language 1 }}</h1>
+            </header>
+        <p class="ons-hero__text">{{ localise "CensusSubhead" .Language 1 }}</p>
+    </div>
+    <div class="ons-hero__circle-image">
+        <img height="558" width="558" srcset="https://cdn.ons.gov.uk/assets/images/census/census-hero-ice-cream-girls-558px-cropped-x1.jpeg 1x, https://cdn.ons.gov.uk/assets/images/census/census-hero-ice-cream-girls-1116px-cropped-x1.jpeg 2x" src="https://cdn.ons.gov.uk/assets/images/census/census-hero-ice-cream-girls-558px-cropped-x1.jpeg" alt="">
+    </div>
+    <svg class="ons-hero__circle-lined" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 243 243" focusable="false" aria-hidden="true">
```

Comment:
This little monster could be pulled out into an icon like the ones in `dp-renderer/assets/templates/icons/*.tmpl`

Although because the frontend shares a namespace with the templates it could go in this frontend's `templates/icons/` as long as there isn't a filename clash with the ones in dp-renderer.